xee/ext.py

```diff
@@ -536,6 +537,44 @@ def _get_primary_coordinates(self) -> List[Any]:
       ]
     return primary_coords
 
+  def _get_tile_from_EE(
+      self, tile_index: Tuple[Any, Union[str, int]]
+  ) -> Tuple[slice, np.ndarray]:
+    """Get a numpy array from EE for a specific bounding box (a 'tile')."""
+    tile_index, band_id = tile_index
+    bbox = self.project(
+        (tile_index[0], 0, tile_index[1], 1)
+        if band_id == 'longitude'
+        else (0, tile_index[0], 1, tile_index[1])
+    )
+    tile_idx = slice(tile_index[0], tile_index[1])
+    target_image = ee.Image.pixelLonLat()
+    return tile_idx, self.image_to_array(
+        target_image, grid=bbox, dtype=np.float32, bandIds=[band_id]
+    )
+
+  def process_coordinate_data(
+      self,
+      total_tile: int,
```

Comment:
I used `total_tile` because I thought it is looks like easily understandable like how much total tiles we have and after then do operation on all the tiles. What's your thought on this?

Reply:
Something declarative like `tile_count` or `num_tiles` is more consistent with other places in our codebase. If you'd like to keep `total_tile`, make sure to pluralize it (e.g. `total_tiles`).

Reply:
Alright, let's proceed ahead with the `tile_count`.